doc/classes/@GlobalScope.xml

```diff
@@ -2752,6 +2799,7 @@
 		<constant name="PROPERTY_HINT_INT_IS_POINTER" value="30" enum="PropertyHint">
 		</constant>
 		<constant name="PROPERTY_HINT_ARRAY_TYPE" value="31" enum="PropertyHint">
+			See [constant PROPERTY_HINT_TYPE_STRING].
```

Comment:
I'm not sure what the difference is between these hints, at first glance they work the same way. Source code search turned up no answers. Perhaps there is a difference in serialization.

Reply:
From looking at the analyzers and docs generators I think `PROPERTY_HINT_ARRAY_TYPE` is supposed to just have the type name of the elements as the hint string.

On the other hand MAKE_RESOURCE_TYPE_HINT (which creates strings of this format) mentions ARRAY_TYPE instead of TYPE_STRING.

Reply:
Yes, I noticed it in #77618 (still needs some work). For now, I've removed the cross-reference.